modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -16,7 +16,8 @@
             "protected-bucket": "{$.meta.buckets.protected.name}",
             "private-bucket": "{$.meta.buckets.private.name}",
             "internal-bucket": "{$.meta.buckets.internal.name}",
-            "file-buckets": "{$.meta.collection.files}"
+            "file-buckets": "{$.meta.collection.files}",
+            "excludeFileTypes": "{[$.meta.collection.meta.excludeFileTypes]}"
```

Comment:
Remove the square brackets should be `"excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"` to avoid a return like `[[".xml"]]